buildman/manager/executor.py

```diff
@@ -170,7 +171,7 @@ def generate_cloud_config(
                 build_uuid=build_uuid,
                 quay_username=quay_username,
                 quay_password=quay_password,
-                manager_hostname=manager_hostname,
+                manager_hostname=manager_hostname.split(":", 1)[0] + ":" + str(DEFAULT_GRPC_SERVER_PORT),
```

Comment:
Can you define this logic outside of the function call arguments and either wrap it in a try/except statement or throw an assertion on it?

Reply:
Moved it outside the function call. It doesn't need a try block since `split` just returns the original string if couldn't find the delimiter.

Reply:
Thank you!